core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -98,5 +158,13 @@
     {{ include_js_file('domain/editor/ChangeObjectFactory.js') }}
     {{ include_js_file('domain/editor/UndoRedoService.js') }}
     {{ include_js_file('domain/summaries/ExplorationSummaryBackendApiService.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
+
+    {{ include_js_file('exploration_editor/EditorServices.js') }}
+    {{ include_js_file('exploration_editor/ExplorationEditor.js') }}
+    {{ include_js_file('exploration_editor/GadgetValidationService.js') }}
```

Comment:
https://docs.angularjs.org/error/$injector/unpr?p0=GADGET_SPECSProvider%20%3C-%20GADGET_SPECS%20%3C-%20gadgetValidationService%20%3C-%20explorationGadgetsService%20%3C-%20explorationStatesService%20%3C-%20routerService

Looks like it's needed for ``GADGET_SPECS``.